dash_docs/chapters/dash_vtk/__init__.py

```diff
@@ -0,0 +1,13 @@
+import sys
+import dash_html_components as html
+
+
+if sys.version_info > (3, ):
+    from .import index
```

Comment:
```suggestion
    from . import index
```
Surprised we don't have linters complaining about this... I guess you're just copying the pattern in `chapters/__init__.py` but ideally we fix that too.